internal/controllers/utils/provision.go

```diff
@@ -360,3 +360,15 @@ func ValidateDefaultInterfaces[T any](data T) error {
 	}
 	return nil
 }
+
+func GetParentPolicyNameAndNamespace(childPolicyName string) (policyName, policyNamespace string) {
```

Comment:
To maintain consistency, this function needs a description above.

Reply:
This function is small and its name clearly conveys its purpose. I didn’t think adding a description was necessary, but I can add one - no big deal.